apps/platform/trpc/routers/convoRouter/convoRouter.ts

```diff
@@ -110,6 +110,44 @@ export const convoRouter = router({
         firstMessageType
       } = input;
 
+      // if there is a send as email identity, check if that email identity is enabled
+      if (sendAsEmailIdentityPublicId) {
+        const emailIdentityResponse = await db.query.emailIdentities.findFirst({
+          where: and(
+            eq(emailIdentities.orgId, orgId),
+            eq(emailIdentities.publicId, sendAsEmailIdentityPublicId)
+          ),
+          columns: {},
+          with: {
+            domain: {
+              columns: {
+                domainStatus: true,
+                sendingMode: true
+              }
+            }
+          }
+        });
+
+        if (!emailIdentityResponse) {
+          throw new TRPCError({
+            code: 'NOT_FOUND',
+            message: 'Send as email identity not found'
+          });
+        }
+
+        if (emailIdentityResponse.domain) {
+          if (
+            emailIdentityResponse.domain.domainStatus !== 'active' ||
+            emailIdentityResponse.domain.sendingMode === 'disabled'
+          ) {
+            throw new TRPCError({
+              code: 'UNPROCESSABLE_CONTENT',
+              message: 'Send as email identity is disabled'
```

Comment:
Change error here to "You cant send from that email address due to a configuration issue. Please contact your administrator or select a different email identity."

Reply:
These error messages are not meant to be seen in frontend anyways, they are for if anyone tries to bypass frontend.
Still changing them